### PR TITLE
Adding remove_files function

### PR DESCRIPTION
**Pull request by rmsamitha**

### CODE REVIEW

File: manifests/remove_files.pp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2016 WSO2, Inc. http://www.wso2.org
```
Comment (chamilad) [inline manifests/remove_files.pp]: Shouldn't this year be 2017?

**Review by chamilad — CHANGES_REQUESTED** [review]
It would be better to add an empty array or a commented entry to wso2/common.yaml as an example on how to set a list of files to be removed.

File: manifests/remove_files.pp (hunk 1 above)
Comment (rmsamitha) [reply manifests/remove_files.pp]: Yes. Fixed with de4c8a09c96553a932628ac48ee09c5c86a698b0

**Comment by rmsamitha** [comment]
Changes done with 2b265881536f837baef3f41b4503029b1d9eb516 and de4c8a09c96553a932628ac48ee09c5c86a698b0

**Review by chamilad — APPROVED** [review]

**Pull request merged**